modules/developer-preview/pages/preview-mode.adoc

```diff
@@ -0,0 +1,87 @@
+
+While the Developer Preview Mode can be enabled using the CLI or REST API, once enabled, it cannot be disabled. 
+
+Note that the commands on this page assume a Ubuntu Linux environment. For the appropriate command-paths, see xref:cli:cli-intro.adoc[CLI Reference].
+
+== How do I know if my cluster is in Developer Preview mode?
+
+If your cluster is in Developer Preview mode, the Admin console displays the following banner:
+image::preview-mode-verify-ui.png[,720,align=left]
+
+You can also verify if Developer Preview mode has been enabled on your cluster by running the `couchbase-cli enable-developer-preview` command with the `--list` flag, and specifying the cluster-address, and the Full Administrator username and password as shown:
+
+----
+/opt/couchbase/bin/couchbase-cli enable-developer-preview \
+--list -c localhost:8091 -u Administrator -p password
+----
+
+If the cluster is not in Developer Preview Mode, the following output is displayed:
+
+----
+Cluster is NOT in developer preview mode
+----
+
+If the cluster _is_ in Developer Preview Mode, the output is as follows:
+
+----
+Cluster is in developer preview mode
+----
+
+== How do I enable the Developer Preview mode? 
+
+To enable the Developer Preview Mode, run the `couchbase-cli enable-developer-preview` command with the `--enable` flag, and specify the cluster-address, and the Full Administrator username and password as shown below. 
+
+----
+/opt/couchbase/bin/couchbase-cli enable-developer-preview \
+--enable -c localhost:8091 -u Administrator -p password
+----
+
+The following notification and prompt are provided:
+
+----
+Developer preview cannot be disabled once it is enabled. If you enter developer preview mode you will not be able to upgrade. DO NOT USE IN PRODUCTION.
+Are you sure [y/n]:
+----
+
+To enter Developer Preview Mode, enter `y` against the prompt, and hit return:
+
+----
+Are you sure [y/n]: y
+----
+
+The following confirmation is now displayed:
+
+----
+SUCCESS: Cluster is in developer preview mode
+----
+
+
+== Developer Preview Features in This Release
+
+The Developer Preview mode in Couchbase Server 6.5 Beta unlocks the following features:
+
+* Collections - These are data containers that can be created within any bucket whose type is either _Couchbase_ or _Ephemeral_.
++
+This allows data-items optionally to be assigned to different collections according to content-type.
++
+For the Developer Preview, collections can be managed by either the _REST API_ or the _CLI_.
+For more information, see xref:developer-preview/collections/collections-overview.adoc[Collections Overview].
+
+* High Data Density
+
+* Cost-based Optimizer
+
+* Index Advisor
+
+* N1QL JavaScript UDFs (Stored Procedures)
```

Comment:
Not just JavaScript UDFs, but inline UDFs also. Maybe rephrase as `N1QL JavaScript UDFs and Stored Procedures`.